### PR TITLE
Introduction Review

### PR DESCRIPTION
**Pull request by oliveiramarcio**
Revised the PT-BR translate of the introduction chapter.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,87 @@
...
+Publicado sobre a licença Creative Commons Attribution 4.0. Você é livre para compartilhar, copiar e redistribuir este livro em qualquer formato, ou remixar e transformá-lo para qualquer propósito (até mesmo comercial). Você deve fornecer os créditos apropriados e disponibilizar um link para a licença
 
 Para mais informações, visite https://creativecommons.org/licenses/by/4.0/
 
 ## Introdução
-Obrigado por obter o Little ASP.NET Core Book! Eu escrevi esse resumido livro com intuito em ajudar desenvolvedores e pessoas interessadas em desenvolvimento web a aprender sobre ASP.NET Core, um novo framework para desenvolver aplicações web e APIs.
+Obrigado por obter o Little ASP.NET Core Book! Eu escrevi esse pequeno livro com o intuito de ajudar desenvolvedores e pessoas interessadas em desenvolvimento web a aprender sobre ASP.NET Core, um novo framework para desenvolver aplicações web e APIs.
 
-O Little ASP.NET Core Book é organizado como um tutorial. Você vai desenvolver uma aplicação do início ao fim e aprender:
+The Little ASP.NET Core Book é organizado como um tutorial. Você desenvolverá uma aplicação do início ao fim e aprenderá:
```
Comment (joaopgrassi) [inline README.md]: Acho que podemos manter o titulo original aqui, mas talvez colocar em aspas?

File: README.md [hunk 2]
```diff
@@ -1,90 +1,87 @@
...
 
-Sinta-se livre em fazer o download se você quiser ver o projeto final ou comparar enquanto você trabalha em seu código.
+Sinta-se livre para fazer download caso queira ver o produto final, ou comparar com seu código enquanto segue os tutoriais do livro.
 
-O livro em si é atualizado frequentemente com correções de erros e novo conteúdo. Se você está lendo um PDF, e-book ou versão impressa, verifique o site oficial ([littleasp.net/book](http://www.littleasp.net/book)) para informa-se se existe uma versão atualizada disponível. Na última página do livro você encontra informações sobre versão e log das alterações.
+O livro em si é atualizado frequentemente com correções de erros e novos conteúdos. Se você está lendo um PDF, e-book, ou versão impressa, verifique o site oficial ([littleasp.net/book](http://www.littleasp.net/book)) para informar-se sobre novas versões atualizadas. Na última página do livro estão informações sobre versões e log das alterações.
 
-### Lendo na sua própria língua
-Graças a alguns fantásticos contribuidores multilíngues, o Little ASP.NET Core Book foi traduzido em outras línguas:
+### Lendo em seu próprio idioma
```
Comment (joaopgrassi) [inline README.md]: Boa!

**Comment by oliveiramarcio** [comment]
Hi! I apologyze! I altered the entire file. In comments on your pull request the text will be so large. But tonight I look my changes on my pull request and then write one by one on the comments of yours.

**Comment by joaopgrassi** [comment]
Nah that's not necessary. I'm reviewing your changes.. almost finishing it. Then after you finish it, I'll merge. For the next one you can start in that way. Let's not throw away the time you put into it :). Good job btw.

File: README.md [hunk 3]
```diff
@@ -1,90 +1,87 @@
...
-Você também tem o grande benefício do _static typing_ do compilador do .NET, que é como ter um _linter_ paranóico ligado o tempo todo. Com isso, fazer algo de errado ou não intencionado em seu código se torna muito mais difícil
+* **Segurança.** A Microsoft leva a segurança à sério, e o ASP.NET Core foi construído pensando em segurança desde o início. ASP.NET Core cuida de diversos itens de segurança, como sanitização de dados de entrada e ataques cross-site request forgery (CSRF - Falsificação de Solicitação Entre Sites), de forma que o desenvolvedor não precise se preocupar com isso. Você também tem o grande benefício da tipagem estática do compilador .NET, que é como ter um linter (ferramenta para verificar erros em códigos fontes) paranoico ligado o tempo todo. Com isso, fazer algo de errado ou não intencionado em seu código se torna muito mais difícil.
```
Comment (joaopgrassi) [inline README.md]: Novamente, eu nao traduziria o termo "static typing" pois e um conceito importante do c# e eh bom as pessoas conhecerem o termo em Ingles caso queiram pesquisar mais sobre. Talves manter em ingles, e colocar em () a traducao em portugues seja uma alternativa.

**Comment by oliveiramarcio** [comment]
Requested adjustments committed.

**Pull request merged**